app/[showStatus]/[showNameAndSeason]/scoring/page.tsx

```diff
@@ -0,0 +1,76 @@
+import ContestantSelector from '../../../components/contestantSelector'
+import { getCompetingEntityList } from "../../../utils/wikiQuery"
+import { getWikipediaContestantDataFetcher } from '../../../utils/wikiFetch'
+import generateListOfContestantRoundLists from '../../../generators/contestantRoundListGenerator'
+
+export const dynamicParams = false
+
+interface showProperties {
+  showNameAndSeason: string,
+  showStatus: string
+}
+
+// Creates routes for scoring
+export function generateStaticParams() {
+  
+    // Necessary Node modules to fetch data
+    const fs = require('fs');
+    const path = require('path');
+    
+    // Based on availability in leagueData
+    const pathToLeagueData = path.join(process.cwd(), 'app', 'leagueData');
+    const shows:Array<showProperties> = [];
+    fs.readdirSync(pathToLeagueData).map((file: string) => {
+      // Needed status for url
+      const { LEAGUE_STATUS } = require(`../../../leagueConfiguration/${file.replace('.js', '.tsx')}`);
```

Comment:
I think it's appropriate to update all the `leagueConfiguration/` files to just be `.js` files... but lets do it in another PR, preferably before this one.

Reply:
Fixed in https://github.com/jjm3x3/AmazingRaceFantasy/pull/169